### PR TITLE
edit-button helper

### PR DESCRIPTION
**Pull request by Hodgy**
Added a helper to create an edit button with the pencil icon. This is based off the remove button that already exists.

### CODE REVIEW

File: views/pulsar/v2/helpers/html.html.twig [hunk 1]
```diff
@@ -505,6 +505,57 @@ Used in html.dropdown to insert lines between items.
...
+data-*    | string | Data attributes, eg: `'data-foo': 'bar'`
+
+#}
+{% macro edit_button(options) %}
+    {% spaceless %}
+        {% import _self as html %}
+        {% import '@pulsar/pulsar/v2/helpers/util.html.twig' as util %}
+
+        {{ html.button(options
+        |exclude('target placement')
```
Comment (Stanton) [inline views/pulsar/v2/helpers/html.html.twig]: Indentation

File: views/pulsar/v2/helpers/html.html.twig [hunk 2]
```diff
@@ -505,6 +505,57 @@ Used in html.dropdown to insert lines between items.
 
 {# -----------------------------------------------------------------------------
 
+# Edit button
+
+Used by other helpers to add the edit icons . Uses data-attributes to
```
Comment (Stanton) [inline views/pulsar/v2/helpers/html.html.twig]: Extra space before .

File: views/pulsar/v2/helpers/html.html.twig [hunk 3]
```diff
@@ -505,6 +505,57 @@ Used in html.dropdown to insert lines between items.
...
+        {{ html.button(options
+        |exclude('target placement')
+        |merge({
+            'class': options.class|default ~ ' edit-button',
+            'type': options.type|default('button'),
+            'data-action': 'edit',
+            'data-action-target': options.target|default,
+            'data-placement': options.placement|default('right'),
+            'data-toggle': 'tooltips',
+            'data-title': options.tooltip|default('Edit this item'),
```
Comment (Stanton) [inline views/pulsar/v2/helpers/html.html.twig]: Alphabetical order

**Comment by Stanton** [comment]
On mobile so apologies for brief notes, looks good!

**Comment by jamesjacobs** [comment]
Wondering in this could be a more generic icon button helper (that takes any icon). Also needs `role="button"` and `aria-label="Edit"`

**Pull request merged**